src/main/java/Car.java

```diff
@@ -70,7 +70,7 @@ public void newLocation(double x, double y, double distanceIn) {
      */
     @Override
     public String toString() {
-        return "Car #:" + id + "\tTime traveled: " + time;
+        return (id + 1) + "\t\t" + String.format("%.1f", time);
```

Comment:
Oh wow that's actually pretty sick. I didn't know that you could do that in Java.